Ginger/GingerCore/ALM/QtestCore.cs

```diff
@@ -0,0 +1,685 @@
+                                        }
+                                    }                                   
+                                }
+                                else
+                                {
+                                    //No matching TC was found for the ActivitiesGroup in QC
+                                    result = "Matching TC's were not found for all " + GingerDicser.GetTermResValue(eTermResKey.ActivitiesGroups) + " in qTest.";
+                                }
+                            }
+                        }
+                    }
+                    else
+                    {
+                        //No matching Test Set was found for the BF in QC
+                        result = "No matching Test Set was found in qTest.";
+                    }
+
+                }
+                if (result == string.Empty)
+                {
+                    result = "Export performed successfully.";
+                    return true;
+                }
+                else
+                {
+                    return false;
+                }
+            }
+            catch (Exception ex)
+            {
+                result = "Unexpected error occurred- " + ex.Message;
+                Reporter.ToLog(eLogLevel.ERROR, "Failed to export execution details to Qtest", ex);               
+                return false;
+            }
+        }
+
+        public void SetTestStepLogStatus(Activity matchingActivity, ref QTestApiModel.TestStepLogResource testStepLog, List<QTestApiModel.StatusResource> statuses)
+        {
+            if (matchingActivity != null)
+            {
+                switch (matchingActivity.Status)
+                {
+                    case Amdocs.Ginger.CoreNET.Execution.eRunStatus.Failed:
+                        testStepLog.Status = statuses.Where(z => z.Name == "Failed").FirstOrDefault();
+                        break;
+                    case Amdocs.Ginger.CoreNET.Execution.eRunStatus.NA:
+                        testStepLog.Status = statuses.Where(z => z.Name == "Not Applicable").FirstOrDefault();
+                        break;
+                    case Amdocs.Ginger.CoreNET.Execution.eRunStatus.Passed:
+                        testStepLog.Status = statuses.Where(z => z.Name == "Passed").FirstOrDefault();
+                        break;
+                    case Amdocs.Ginger.CoreNET.Execution.eRunStatus.Skipped:
+                        testStepLog.Status = statuses.Where(z => z.Name == "Unexecuted").FirstOrDefault();
+                        break;
+                    case Amdocs.Ginger.CoreNET.Execution.eRunStatus.Pending:
+                        testStepLog.Status = statuses.Where(z => z.Name == "Deffered").FirstOrDefault();
+                        break;
+                }
+            }
+            else
+            {
+                testStepLog.Status = statuses.Where(z => z.Name == "Unexecuted").FirstOrDefault(); ;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this empty statement.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)